packages/protocol/test/identity/federatedattestations.ts

```diff
@@ -20,33 +21,290 @@ contract('FederatedAttestations', (accounts: string[]) => {
   let federatedAttestations: FederatedAttestationsInstance
   let registry: RegistryInstance
 
-  const caller: string = accounts[0]
-  // const phoneNumber: string = '+18005551212'
-  // const phoneHash: string = getPhoneHash(phoneNumber)
+  const account1 = accounts[0]
+  const issuer1 = accounts[1]
+  const phoneNumber: string = '+18005551212'
+  const pnIdentifier1 = getPhoneHash(phoneNumber)
+  const nowUnixTime = Math.floor(Date.now() / 1000)
 
   beforeEach('FederatedAttestations setup', async () => {
     accountsInstance = await Accounts.new(true)
     federatedAttestations = await FederatedAttestations.new(true)
     registry = await Registry.new(true)
     await accountsInstance.initialize(registry.address)
     await registry.setAddressFor(CeloContractName.Accounts, accountsInstance.address)
+    await registry.setAddressFor(
+      CeloContractName.FederatedAttestations,
+      federatedAttestations.address
+    )
     await federatedAttestations.initialize(registry.address)
   })
 
   describe('#initialize()', () => {
     it('TODO ASv2', async () => {
       // TODO ASv2
-      assert(caller)
       assert(federatedAttestations)
     })
   })
 
   describe('#lookupAttestations', () => {
```

Comment:
We can also add a test case for calling `lookupAttestations` on an identifier that isn't yet registered

Reply:
Similarly for `lookupIdentifiersByAddress` we can pass in an account that isn't yet registered